marlowe/cli-doc/extended-tutorial.md

```diff
@@ -0,0 +1,250 @@
+# Marlowe CLI Tutorial: Extended Example
+
+This example for using `marlowe-cli` runs a Marlowe three-step contract on `testnet`.
+
+1.  The contract is initially funded with 3 ADA.
+2.  The sole party then deposits 10 ADA.
+3.  The contract then permits 5 ADA to be payed back to the party.
+4.  Finally, the contract closes and the remaining 8 ADA is withdrawn.
+
+Here is the contract in Marlowe format;
+
+<table>
+<tr>
+<td>
+<ol>
+<li value="0">The contract is initially funded with 3 ADA.
+<li value="1">The sole party then deposits 10 ADA.
+<li value="2">The contract then permits 5 ADA to be payed back to the party.
+<li value="3">Finally, the contract closes and the remaining 8 ADA is withdrawn.
+</ol>
+<td>
+<td rowspan="2">
+<img alt="A Three-Step Marlowe Contract" src="extended-example.png"/>
+</td>
+</tr>
+<tr>
+<td>
+<pre>
+When
+    [Case
+        (Deposit
+            (PK "0a11b0c7e25dc5d9c63171bdf39d9741b901dc903e12b4e162348e07")
+            (PK "0a11b0c7e25dc5d9c63171bdf39d9741b901dc903e12b4e162348e07")
+            (Token "" "")
+            (Constant 10)
+        )
+        (Pay
+            (PK "0a11b0c7e25dc5d9c63171bdf39d9741b901dc903e12b4e162348e07")
+            (Account (PK "0a11b0c7e25dc5d9c63171bdf39d9741b901dc903e12b4e162348e07"))
+            (Token "" "")
+            (Constant 5)
+            Close
+        )]
+    90000000 Close
+</pre>
+</td>
+</tr>
+</table>
+
+
+## 1. Select the network.
+
+Make sure that `marlowe-cli`, `cardano-cli`, and `jq` have been installed on the path for the `bash` shell. Set the environment variable `CARDANO_NODE_SOCKET_PATH` to point to the location of the socket for the `cardano-node` service. In this tutorial, we use the public `testnet`:
+
+    NETWORK=testnet
+    MAGIC="--testnet-magic 1097911063"
+    export CARDANO_NODE_SOCKET_PATH=$PWD/$NETWORK.socket
+
+
+## 2. Select the wallet.
+
+Select a wallet for use in this tutorial and specify the files with the signing and payment keys. The address of this wallet is stored in the environment variable `ADDRESS_P`.
+
+    PAYMENT_SKEY=payment.skey
+    PAYMENT_VKEY=payment.vkey
```

Comment:
suggesting convention I saw in cardano docs. 
```suggestion
    PAYMENT_SKEY=path/to/payment.skey
    PAYMENT_VKEY=path/to/payment.vkey
```

Reply:
Fixed in 70b73b0f9cee949c16716ccd93635745080999b5.